### PR TITLE
testing cleanup no tix

### PR DESCRIPTION
**Pull request by RoriCremer**
general refactor and cleanup for testing

### CODE REVIEW

File: src/test/java/bio/terra/fixtures/FlightStates.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+// CHECKSTYLE:OFF
```
Comment (thathert) [inline src/test/java/bio/terra/fixtures/FlightStates.java]: How come?

File: src/test/java/bio/terra/fixtures/FlightStates.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+public class FlightStates {
+
+    public static final String testFlightId = "test-flight-id";
+    public static final Timestamp submittedTime = Timestamp.from(Instant.now());
+    public static final String submittedTimeFormatted = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS")
+            .format(submittedTime);
+    public static final Timestamp completedTime = Timestamp.from(Instant.now());
+    public static final String completedTimeFormatted = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS")
+            .format(completedTime);
+    public static final StudySummaryModel studySummary = new StudySummaryModel()
```
Comment (thathert) [inline src/test/java/bio/terra/fixtures/FlightStates.java]: It'd be useful to have a `StudyFixtures` class alongside this one that can generate Study summaries and Study request models.

File: src/test/java/bio/terra/fixtures/FlightStates.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                .name("Minimal")
+                .description("This is a sample study definition");
+    public static final StudySummaryModel minimalStudySummary = new StudySummaryModel()
```
Comment (thathert) [inline src/test/java/bio/terra/fixtures/FlightStates.java]: I think both of these can be removed and we can use `StudyFixtures.makeStudySummary()` in the the methods below.

**Review by thathert — CHANGES_REQUESTED** [review]
Looks good, one comment about adding StudyFixtures

**Review by thathert — APPROVED** [review]
LGTM

File: src/test/java/bio/terra/fixtures/FlightStates.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import java.time.Instant;
+import java.util.Optional;
+
+
+public final class FlightStates {
+    private FlightStates() {}
+
+    public static final String testFlightId = "test-flight-id";
+    public static final Timestamp submittedTime = Timestamp.from(Instant.now());
+    public static final String submittedTimeFormatted = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS")
```
Comment (ahaessly) [inline src/test/java/bio/terra/fixtures/FlightStates.java:21]: is this supposed to be testing the date format? if not, should you use
    @Autowired
    private SimpleDateFormat modelDateFormat;

**Review by ahaessly — APPROVED** [review]
just one q

**Pull request merged**